docs/integrations/sources/jira-migrations.md

```diff
@@ -1,5 +1,31 @@
 # Jira Migration Guide
 
+## Upgrading to 2.0.0
+
+Note: this change is only breaking if you are using the `Issues`, `Boards Issues` or `Sprint Issues` streams in Incremental Sync mode.
+
```

Comment:
We're continuously striving to enhance the quality and reliability of our connectors at Airbyte. As part of our commitment to delivering exceptional service, we are transitioning source-jira from the Python Connector Development Kit (CDK) to our innovative low-code framework. This is part of a strategic move to streamline many processes across connectors, bolstering maintainability and freeing us to focus more of our efforts on improving the performance and features of our evolving platform and growing catalog. However, due to differences between the Python and low-code CDKs, this migration constitutes a breaking change.